src/java/generator/MapGenerator.java

```diff
@@ -39,15 +43,18 @@ public static void main(String[] args) throws ExecutionException, InterruptedExc
 				
 			}
 		} catch (ArrayIndexOutOfBoundsException | NumberFormatException e) {
-			System.out.println("Usage: generator [targetFolder] [seed] [expectedVersion] (mapName)");
-			
+			throw new IllegalArgumentException("\nUsage: generator [targetFolder] [seed] [expectedVersion] (mapName)");
```

Comment:
Hmm, no need to throw an IllegalArgumentException, that will just give the user a huge stack trace, revert that please.
We should be printing the actual exception that caused this though. (actually only the parameter parsing should be in the try catch block, the rest shouldn't)